lib/common.js

```diff
@@ -1025,7 +1025,15 @@ function createTableFromArray(headers, fields, values) {
 
   _.each(values, function(t) {
     var vals = _.map(fields, function(f) {
-      return t[f] ? t[f] : '';
+      if (t[f] && t[f] !== true) {
```

Comment:
I will implement that this way:

``` javascript
      if (t[f] === true) {
        return i18n._('Yes') ;
      } else if (t[f] === false) {
        return i18n._('No') ;
      }
      return !!t[f] ? t[f] : '';
```

Or even simpler without using natural language

```javascript
   if(ts[f]===undefined && ts[f]===null){
      return '';
   }
   return ts.toString();
```